README.md

```diff
@@ -1,6 +1,9 @@
 # ocaml-cordova-plugin-firebase
 
-Binding to [cordova-plugin-firebase](https://github.com/arnesson/cordova-plugin-firebase)
+Binding to
+[cordova-plugin-firebase](https://github.com/arnesson/cordova-plugin-firebase),
```

Comment:
This OCaml library is not a binding to this Cordova plugin, isn't it?

Reply:
OK, I see why you write this after reading https://github.com/besport/ocaml-cordova-plugin-firebase/issues/4#issuecomment-901900133.

Reply:
However, if I am not mistaken, there are other links in the README file that need to be updated.